src/ol_infrastructure/applications/ocw_site/__main__.py

```diff
@@ -211,3 +212,225 @@
         "policy": s3_bucket_iam_policy.name,
     },
 )
+
+#################
+# Fastly Config #
+#################
+
+# Website Storage Bucket
+website_storage_bucket_fqdn = "ocw-website-storage.s3.us-east-1.amazonaws.com"
+
+for purpose in ["draft", "live"]:
+    website_bucket_fqdn = (
+        f"ocw-content-{purpose}-{stack_info.env_suffix}.s3.us-east-1.amazonaws.com"
+    )
+
+    servicevcl_backend = fastly.ServiceVcl(
+        f"ocw-{purpose}-{stack_info.env_suffix}",
+        backends=[
+            fastly.ServiceVclBackendArgs(
+                address=website_bucket_fqdn,
+                name="WebsiteBucket",
+                override_host=website_bucket_fqdn,
+                port=443,
+                request_condition="not course media or old akamai",
+                ssl_cert_hostname=website_bucket_fqdn,
+                ssl_sni_hostname=website_bucket_fqdn,
+                use_ssl=True,
+            ),
+            fastly.ServiceVclBackendArgs(
+                address=website_storage_bucket_fqdn,
+                name="OCWWebsiteStorageBucket",
+                override_host=website_storage_bucket_fqdn,
+                port=443,
+                request_condition="is old Akamai file",
+                ssl_cert_hostname=website_storage_bucket_fqdn,
+                ssl_sni_hostname=website_storage_bucket_fqdn,
+                use_ssl=True,
+            ),
+        ],
+        comment="",
```

Comment:
Is this a necessary parameter? Maybe leave it out if we're not going to populate it.